### ISSUE TITLE
[Minutes] SG Minutes 2022-10-19

### ISSUE THREAD

**Issue by cwilso** [whatwg/sg#201]
Woof, been a while since we remembered to post these, so sorry.

### 2022-10-19
Review open PRs:

- Discussed/approved https://github.com/whatwg/whatwg.org/pull/402 

Review open issues (https://github.com/whatwg/sg/issues):

- https://github.com/whatwg/sg/issues/63 - bring up to legal
- https://github.com/whatwg/sg/issues/194 - bring up to legal

### 2022-10-05
Review open PRs:

- None!!

Review open issues (https://github.com/whatwg/sg/issues):

- Nothing new, so just chipping away at backlog:
- https://github.com/whatwg/sg/issues/196 - Anne to file PR
- https://github.com/whatwg/sg/issues/194 - Needs tooling, on hold
- https://github.com/whatwg/sg/issues/63 - Needs work before proposing to legal

### 2022-09-21
Review open PRs:

- https://github.com/whatwg/sg/pull/158:  not waiting on legal anymore, just waiting on final cleanup. Request [change to 170](https://github.com/whatwg/participate.whatwg.org/pull/170#pullrequestreview-1115786860) as part of this
- https://github.com/whatwg/sg/pull/59  Ensure this lines up with current license text, remove YEAR.

Review open issues (https://github.com/whatwg/sg/issues):

- Ran out of time
